.vscode/launch.json

```diff
@@ -1,25 +1,9 @@
 {
   "version": "0.2.0",
   "configurations": [
-    {
-      "name": "Debug examples/fastapi",
-      "type": "python",
-      "request": "launch",
-      "module": "uvicorn",
-      "args": [
-        "main:app",
-        "--app-dir",
-        "${workspaceFolder}/examples/fastapi/arcade_example_fastapi",
-        "--port",
-        "8002"
-      ],
-      "jinja": true,
-      "justMyCode": true,
-      "cwd": "${workspaceFolder}/examples/fastapi/arcade_example_fastapi"
-    },
     {
       "name": "Debug `arcade workerup --no-auth`",
-      "type": "python",
+      "type": "debugpy",
```

Comment:
`type: python` is being deprecated in favor of `type: debugpy`

See https://devblogs.microsoft.com/python/python-in-visual-studio-code-february-2024-release/